Sources/Client/Client/ClientLogger.swift

```diff
@@ -216,7 +216,7 @@ public final class ClientLogger {
     private var startTime: CFTimeInterval
     private let level: Level
     
-    private var measuredTasksStart: Atomic<[UUID: (start: Date, description: String)]> = .init([:])
+    private var measuredTasksStart: Atomic<[UUID: (start: CFTimeInterval, description: String)]> = .init([:])
```

Comment:
why this change?

Reply:
It's more precisely and easy to manage. Always in seconds.